gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversalTest.java

```diff
@@ -71,4 +80,89 @@ public void graphTraversalShouldHaveMethodsOfAnonymousGraphTraversal() {
             }
         }
     }
+
+    @Test
+    public void graphTraversalMethodsShouldGenerateCorrespondingBytecode() throws Exception {
+        final Random random = new Random();
```

Comment:
Lets keep it un-seeded. If it breaks down the road, we learn something. If not, then its good.